### PR TITLE
Bug Fixes: Fixed incorrect reactive element paths in views + added missing react-native-svg which is required for react native elements

### PR DESCRIPTION
**Pull request by lakhbawa**
…ssing react-native-svg which is required for react native elements

### CODE REVIEW

**Review by khushal87 — COMMENTED** [review]
The app uses `yarn` therefore please update your dependencies such that it aligns the yarn convention by having `yarn.lock` instead of `package-lock.json`

File: tsconfig.json [hunk 1]
```diff
@@ -1,51 +1,37 @@
 {
```
Comment (khushal87) [inline tsconfig.json:1]: Why did you remove all the comments and entries from here? Please add it back. This is not right according to me.

Comment (lakhbawa) [reply tsconfig.json:1]: i guess, it was removed by npm, let me do it for you

**Comment by lakhbawa** [comment]
> The app uses `yarn` therefore please update your dependencies such that it aligns the yarn convention by having `yarn.lock` instead of `package-lock.json`

ok,

File: src/views/text.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Linking, StyleSheet, View } from 'react-native';
 import { Text, Button } from 'react-native-elements';
-import theme from 'react-native-elements/src/config/theme';
+import theme from 'react-native-elements/dist/config/theme';
```
Comment (arpitBhalla) [inline src/views/text.tsx:4]: Why we need to change this? It was working till now, just asking..

Comment (lakhbawa) [reply src/views/text.tsx:4]: did not work for me, pls try cloning the project from scratch

Comment (arpitBhalla) [reply src/views/text.tsx:4]: I did, working fine for me, I am not sure what you did

Comment (arpitBhalla) [reply src/views/text.tsx:4]: Working Fine for me without this change

Comment (lakhbawa) [reply src/views/text.tsx:4]: I am not sure, I had to change it, to get it working, It was continuously giving error until i made those changes

Comment (lakhbawa) [reply src/views/text.tsx:4]: maybe it's just me because it's my first time using the react-native

Comment (arpitBhalla) [reply src/views/text.tsx:4]: Great! Keep Learing

Comment (arpitBhalla) [reply src/views/text.tsx:4]: Great! Keep Learning

Comment (arpitBhalla) [reply src/views/text.tsx:4]: Yeah, You are right,

**Comment by flyingcircle** [comment]
Yep, seems to work fine without any additional changes. I don't see any need for these.
